urdf_parser_plugin/include/urdf_parser_plugin/parser.h

```diff
@@ -37,7 +37,7 @@
 #ifndef URDF_PARSER_PLUGIN_H
 #define URDF_PARSER_PLUGIN_H
 
-#include <urdf_model/model.h>
+#include <urdf/urdfdom_compatibility.h>
```

Comment:
This also means you need to have `urdf` as a `run_depend`. If we were using package xml version 2, it would be a `build_export_depend`. Basically, since there is no separate `-dev` package `urdf` needs to be installed at "run time" so other packages would be able to build against it. This is likely handled implicitly as a transitive dependency of one of the other `run_depend`'s.